public/video-ui/src/components/VideoAssetAdd/formComponents/VideoAssetUrl.js

```diff
@@ -6,7 +6,7 @@ import React from 'react';
 export default class VideoAssetUrl extends React.Component {
 
   onUpdateAssetUrl = (e) => {
-    const newVersion = this.props.video.activeVersion === undefined ? 0 : this.props.video.activeVersion + 1;
+    const newVersion = this.props.video.activeVersion === undefined ? 0 : Math.max(...this.props.video.assets.map(x => x.version)) + 1;
```

Comment:
Say you have asset versions 1, 2 and 3 and you roll back the `activeVersion` to 2, then the new version generated would be 3. This causes a version conflict.